index.html

```diff
@@ -149,142 +167,206 @@ <h2>Multi-Post Stories</h2>
-          </a>
+          <img src="images/Ellipse_3.png" alt="CSS" />
+          <a href="">CSS</a>
         </li>
       </ul>
-      <a class="btn resume-btn" href="#"> Get my resume</a>
-    </div>
-    <ul class="knowledges">
-      <li class="knowledge">
-        <h2 class="kn_1">
-          <span>Languages</span>
-          <span>
-            <img src="images/arrow_bottom.png" alt="arrow_bottom">
-          </span>
-        </h2>
-        <ul class="languages">
-          <li>
-            <img src="images/Ellipse1.jpg" alt="JavaScript" />
-            <a href="#">JavaScript</a>
-          </li>
-          <li>
-            <img src="images/Ellipse1(1).jpg" alt="HTML" />
-            <a href="#">HTML</a>
-          </li>
-          <li>
-            <img src="images/Ellipse1(2).jpg" alt="CSS" />
-            <a href="#">CSS</a>
-          </li>
-        </ul>
-      </li>
-      <li class="knowledge">
-        <h2 class="kn_1">
-          <span>Frameworks</span>
-          <span>
-            <img src="images/arrow_right.png" alt="arrow_right">
-          </span>
-        </h2>
-      </li>
-      <li class="knowledge">
-        <h2 class="kn_1">
-          <span>Skills</span>
-          <span>
-            <img src="images/arrow_right.png" alt="arrow_right">
-          </span>
-        </h2>
-      </li>
-    </ul>
-  </section>
-
-  <section class="contact-form">
-    <div class="title-p">
-      <h1 class="contact-me">Contact me</h1>
-      <p class=p2>If you have an application you are interested in developing,
-         a feature that you need built or a project that needs coding.
-         I’d love to help with it </p>
-    </div>
-    <form  class="form" action="https://formspree.io/f/xbjwakbe" method="post">
-      <ul>
-        <li class="text">
-          <input required type="text" id="name" name="user_name" maxlength="30" placeholder="Write your name"/>
+    </li>
+    <li class="knowledge">
+      <h2 class="kn_1">
+        <span>Skills</span>
+        <span>
+          <img src="images/arrow_right.png" alt="arrow_right">
+        </span>
+      </h2>
+      <ul class="frameworks">
+        <li>
+          <img src="images/Ellipse_1.png" alt="JavaScript" />
+          <a href="">JavaScript</a>
         </li>
-        <li class="text">
-          <input required type="email" id="mail" name="user_email" placeholder="Write your email" />
+        <li>
+          <img src="images/Ellipse_2.png" alt="HTML" />
+          <a href="">HTML</a>
         </li>
-        <li class="text">
-          <textarea required id="msg" name="user_message" maxlength="500" placeholder="Write your message here"></textarea>
+        <li>
+         <img src="images/Ellipse_3.png" alt="CSS" />
+          <a href="">CSS</a>
         </li>
-      </ul> 
-      <input class="butt" type="submit" value="Get It Touch">
+      </ul>
+    </li>
+  </ul>
+</section>
+  <footer id="contact" class="contact">
+    <div class="contact-contact-container">
```

Comment:
there is some gray background color between the `about me` container and the `form` container, please make sure to fill that gray space with white color: 
<img width="767" alt="Screenshot 2022-04-01 at 12 40 50" src="https://user-images.githubusercontent.com/77038610/161227901-eb89025b-c222-4c26-b78d-7a88be5f5fa7.png">